app/jobs/disability_claim/document_upload.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
```

Comment:
Should we move these to a `app/workers` directory to be consistent with Sidekiq convention or just leave in `jobs` to match Rails convention?